api/src/certification/session-management/domain/usecases/finalize-session.js

```diff
@@ -119,7 +119,10 @@ async function _removeAbortReasonFromCompletedCertificationCourses({
         sessionCertificationCourse.isCompleted()
       ) {
         sessionCertificationCourse.unabort();
-        await certificationCourseRepository.update({ certificationCourse: sessionCertificationCourse });
+        // do not use transaction so throwing error will not revert the session unabort
+        await certificationCourseRepository.updateWithoutTransaction({
```

Comment:
Le nouveau parametre me semble suffisamment explicite, je propose de ne pas mentionner un detail technique dans le nommage d'une fonction.

```suggestion
        await certificationCourseRepository.update({ noTransaction = true,
```

Reply:
Quelle est la différence avec ajouter ce détail technique en paramètre ?

Reply:
Une bonne pratique courante de ne pas porter sur un nom de methode un detail d'implementation et de rester au niveau de l'intention.

Un comme comme ne pas ecrire `const identifiersSet = new Set([1, 2])`, tu vas plutot soit juste mettre `identifiers` voir `uniqueIdentifiers` comme nom.